src/uct/ib/ud/base/ud_iface.c

```diff
@@ -1041,6 +1043,19 @@ void uct_ud_iface_progress_disable(uct_iface_h tl_iface, unsigned flags)
     uct_base_iface_progress_disable(tl_iface, flags);
 }
 
+void uct_ud_iface_vfs_refresh(uct_iface_h iface)
+{
+    uct_ud_iface_t *ud_iface = ucs_derived_of(iface, uct_ud_iface_t);
+
+    ucs_vfs_obj_add_ro_file(ud_iface, ucs_vfs_show_primitive,
+                            &ud_iface->rx.available, UCS_VFS_TYPE_INT,
+                            "rx/available_credits");
```

Comment:
I'd go with `rx_available` instead. IMO we don't need a deeper hierarchy here,  + consistency with RC.
 @yosefe  WDYT?

Reply:
ok